src/components/CheckboxInput/CheckboxInput.js

```diff
@@ -114,17 +121,35 @@ export const CheckboxInput = ({
 }
 
 CheckboxInput.propTypes = {
+  /** low level prop used only by the form engine */
   formTouched: PropTypes.bool,
-  name: PropTypes.string.isRequired, // must be unique
+  /** Name of the field */
+  name: PropTypes.string.isRequired,
+  /** Required data-tid used as a unique id for targeting test selectors */
   'data-tid': PropTypes.string.isRequired,
+  /** Optionally sets a default value for the checkbox */
   initialValue: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
+  /** Allows for brute force setting of whether checked or not. Usually,
```

Comment:
@stephaniel96 This is the compromise of this effort is to expose `checked` and have it take precedence over other means. Otherwise, we should let the checkboxes be dictated by the component itself.

I tried other means for a good couple of hours and this was the only approach I could get to work so 🤷‍♂